go.mod

```diff
@@ -48,11 +48,11 @@ require (
 	github.com/pingcap/gofail v0.0.0-20181217135706-6a951c1e42c3
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20190110035000-d4fe6b336379
+	github.com/pingcap/log v0.0.0-20190129103703-6afc545b8868
```

Comment:
Because this PR need to add the pingcap/log, but now we use pingcap/log#2 to replace pingcap/log. After the  pingcap/log#2 merging, I will remove line90.